### DISCUSSION TITLE
Installation/configuration with yum

### DISCUSSION THREAD

**Discussion by JasGawera** [randombit/botan#3303] [Q&A]
I'm looking to use the botan library.
I've installed the library using 
`
sudo yum install botan2 botan2-devel
`
This installed version 2.12.1 
When I try to use the library, I get compiler errors due to it not finding the include files.
I see the files have been installed under
/usr/include/botan-2/botan
To get it to work, I had to create a symbolic link at
/usr/include/botan pointing to the above directory.

Is this the expected way to install and use the library?
Or have I done something wrong?
Or is this a bug?

Thanks

**Comment by reneme** [comment]
To the best of my knowledge, we (the library developers) don't maintain the distribution packages ourselves. Assuming that you are using Botan on some flavour of RedHat, you could open a ticket here: https://bugz.fedoraproject.org/botan2

The package comes with a PkgConfig module though, so I would assume that it installs the headers in that location by design. Are you using some build system like CMake to find dependencies in your project?

**Reply by randombit** [reply]
This looks to me as working as intended. The idea here is to allow parallel installation of multiple major versions. The intention is that in your compilation process you would set `-I/usr/include/botan-2` and then in your application software `#include <botan/foo.h>`
